lib/src/services/music.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:nyxx_lavalink/nyxx_lavalink.dart';
+import 'package:running_on_dart/running_on_dart.dart';
+import 'package:running_on_dart/src/util.dart';
+
+/// The address of the lavalink running server to connect to.
+String serverAddress = getEnv('LAVALINK_ADDRESS', 'localhost');
```

Comment:
Can you add these environment variable to the README and move them to the settings file like all other environment variables?

Reply:
done!